ui/app/AppLayouts/Profile/Sections/AdvancedContainer.qml

```diff
@@ -20,4 +20,49 @@ Item {
         font.weight: Font.Bold
         font.pixelSize: 20
     }
+
+    RowLayout {
+        id: browserTabSettings
+        anchors.top: element7.bottom
+        anchors.topMargin: 20
+        anchors.left: parent.left
+        anchors.leftMargin: 24
```

Comment:
Usually, anchors inside a RowLayout give warnings. The goal of Rows is that they get managed automatically after the other row. If you prefer using anchors, then I'd recommend switching to `Item`

Reply:
Nvm, I messed up